.buildkite/release-pipelines/update-metadata-on-app-store-connect.yml

```
@@ -23,7 +23,7 @@ steps:
       bundle exec fastlane run configure_apply
 
       echo '--- :shipit: Update Release Notes and Other App Store Metadata'
-      bundle exec fastlane update_metadata_on_app_store_connect skip_confirm:true
+      bundle exec fastlane update_metadata_on_app_store_connect skip_confirm:true with_screenshots:"$WITH_SCREENSHOTS"
```

Just to be on the safe side but also to make the transition and synchronization with the deployment of https://github.a8c.com/Automattic/wpcom/pull/196883 easier.

```suggestion
      bundle exec fastlane update_metadata_on_app_store_connect skip_confirm:true with_screenshots:"${WITH_SCREENSHOTS:-false}"
```